### DISCUSSION TITLE
Give your strings context with StringSyntaxAttribute

### DISCUSSION THREAD

**Discussion by giscus** [linkdotnet/Blog.Discussions#37] [General]
# Give your strings context with StringSyntaxAttribute

Strings are one of the most universal data types. We use them for URLs or regular expressions or even to define some date. With .NET 7 we have a new way of giving those strings a bit of meaning. Meet StringSyntaxAttribute.
I also show you a way how to use them in .NET 6 and earlier.

https://steven-giesel.com/blogPost/d2b2fc18-ca4c-4879-b87f-a1d36f435805

**Comment by 274188A** [comment]
https://github.com/linkdotnet/BlogExamples/blob/main/StringHightlighting =>404

**Reply by linkdotnet** [reply]
Thanks, corrected. Should work now.
